content/toolsAndResources/nge/ngeBlocks.md

```diff
@@ -0,0 +1,1387 @@
+
+#### Outputs
+- **uv** is the geometry after computing normals.
+
+![Merge node](/img/tools/nge/mergeNode.jpg)
+
+### Merge
+A common operation when creating procedural geometry is to merge two or more different meshes. This node will merge up to five geometry inputs into one using a simple merge. This node does not do any type of boolean operation and does not change the vertex lists or UV coordinates at all. If multiple meshes are merged which have different material IDs, a multimaterial will be created and assigned to keep the original material breaks of each geometry.
+
+#### Advanced
+- **Evaluate context** is used for optimization in the graph when using successive iterable nodes. If this property is set to true, each iteration in this node will also calculate all looped context feeding this node. If there are several iterable nodes connected in sequence, the calculations become exponential with every iteration of the current node calculating every iteration of the previous node back to the beginning of the chain. This can cause a spike in process time in the graph. Disabling evaluate context will prevent nodes from initiating evaluations of previous context each iteration which will greatly speed up the graph. However, since there may be a case where it is desireable to re-evaluate context with each iteration of a node, this parameter is available to the user and defaults to true. If a graph seems to be taking too long to process, check iterable nodes to see if disabling the evaluation of context will help improve performance.
+
+#### Inputs
+- **geometry0** can optionally be connected to a node providing a Geometry type.
+- **geometry1** can optionally be connected to a node providing a Geometry type.
+- **geometry2** can optionally be connected to a node providing a Geometry type.
+- **geometry3** can optionally be connected to a node providing a Geometry type.
+- **geometry4** can optionally be connected to a node providing a Geometry type.
+
+#### Outputs
+- **output** is the merged geometry made from those available through connectioned inputs.
+
+![Optimize node](/img/tools/nge/optimizeNode.jpg)
+
+### Optimize
+There are times when a geometry will have unwelded vertices where mutliple vertices occupy the same position. This is common when using glTF files as the specification states that vertices that lie on a UV seam should be split for optimization reasons. If working with geometry that has unwelded vertices that need to be merged, use the optimize node. This node does a simple operation that merges any vertices that share the same position within a certain epsilon. This operation is destructive in that it creates a new triangle list and deletes all vertex parameters other than positions. This means that any mesh that has been optimized will need new normals, tangents, and vertex colors. Typically, the first node connected after an Optimize node would be a ComputeNormals node.

```

Comment:
nit: mutliple => multiple

Reply:
fixed in https://github.com/BabylonJS/Documentation/pull/963/commits/5395b92d4e3c0f28c82faf008026b09b521801ce